### ISSUE TITLE
Different recipients for SNV and SV verification emails

### ISSUE THREAD

**Issue by bjhall** [Clinical-Genomics/scout#1873]
At the moment it is only possible to set a single email address for verification emails of an institute ("sanger_recipient"). In our case SNVs are verified by the sanger facility whereas SVs (only CNVs really) are verified by the array facility.

It would be nice to be able to set different email addresses for them (and possibly STRs etc as well). I can probably find some time to work on this myself, but I would like to get some input before I start working on it!

**Comment by dnil** [comment]
New fun features are always welcome!

Sanger recipient is supposed to be an array, where It is possible to set multiple recipients for a single institute? Let us know if we missed that somewhere!

We did have this discussion internally at clinical genetics at least, but it sort of fizzled out when it was realised that sometimes the most practical way of testing an SV is actually just running an MLPA kit (which was done by the same team as Sanger for SNVs), breakpoint PCR, breakpoint Sanger sequencing, qPCR or by all means different available arrays. Getting the coordinates and then directing the info to the team best equipped to handle it seems more important that automatically sending it to a particular site. Perhaps an assignment system, where recipients can call dibs on a particular verification would be helpful?

**Comment by dnil** [comment]
(Also when accepting high quality SNV calls without independent orthogonal method verification, the total number became very manageable.)

**Comment by bjhall** [comment]
Unfortunately klingen in Lund does not do MLPA or any customized PCR other than for SNVs/small indels, so at the moment we are forced to verify on array...

It is indeed possible to add multiple recepients, but I think there is a fear that the recipients will be confused. But I completely understand your situation, it is not always clear what the best verification method is. Maybe it would be better if the array and sanger facilities shared an inbox and somehow communicated with each other... I'll take this discussion internally once more.

**Comment by dnil** [comment]
And we could potentially help them with that, allowing those teams to assign themselves/"reply" to verification orders, e.g. by having them modify the status to say "under verification" as one of the users from the appropriate institute?

**Comment by bjhall** [comment]
Yes that's a good idea. I'll suggest it and see what people think.
